client/gutenberg/extensions/todo/editor.scss

```diff
@@ -23,17 +23,17 @@ $white: #fff;
 
 $color-accent-green: #31843f;
 
-.wp-editor-a8c-todo {
+.wp-block-a8c-todo {
 	margin-bottom: 20px;
 
 	// This would be ideal but it's being overridden.
-	.wp-editor-a8c-todo-list {
+	.wp-block-a8c-todo__list {
```

Comment:
Consider not nesting these (`.wp-block-a8c-todo .wp-block-a8c-todo__list`) since they both have enough specificity already.

Reply:
At a glance, these classes look overly-specific. However, given some of the comments about specificity, I'm inclined to give the original author the benefit of the doubt that everything is just as specific as it needs to be.

There's added complexity of testing this in the context of p2/o2, which I'm not sure how to approach at this point.